dev-support/hbase_docker.sh

```diff
@@ -159,4 +159,4 @@ done
 echo "Successfully built ${IMAGE_NAME}."
 
```

Comment:
Oh no, I didn't know this script existed. I probably broke it with the improvements that went in with HBASE-23945.